kokoro/config/test/image_lists.gcl

```diff
@@ -15,149 +15,149 @@ local template _distro {
 // DEB Linux distros. (Do not modify this comment.)
 bookworm_aarch64 = _distro {
   release = [
-      'debian-12-arm64',
```

Comment:
Can we do something like this instead:
```
local template _distro {
  // Shared image project.
  image_project = external
  // List of release images.
  release_images = external
  // The representative images to be used in presubmit (defaults to none).
  // Presubmits usually smoke test only the oldest and newest OS families from
  // a certain OS to speed up presubmit tests and reduce flakiness. The full
  // set will be tested by the nightly jobs.
  presubmit_images = []
  // List of release image specs.
  release = map(lambda i_: cond(match(i_, '[:=]'), i_, "%image_project%:%i_%"), release_images)
  // The representative image specs to be used in presubmit.
  presubmit = map(lambda i_: cond(match(i_, '[:=]'), i_, "%image_project%:%i_%"), presubmit_images)
}
```
and then just use something like:
```
bookworm_aarch64 = _distro {
  image_project = 'debian-cloud'
  release_images = [
      'debian-12-arm64',
  ]
  presubmit_images = [
      'debian-12-arm64',
  ]
}
```
?

Reply:
Several distros are tested on images living in multiple projects. So the structure would have to be somewhere different. That being said, What I'd really like to do is remove `image_lists.gcl` and make `project.yaml` the source of truth.